esy.json

```diff
@@ -1,8 +1,20 @@
 {
+  "esy": {
+    "build": "make build",
```

Comment:
Can we call dune directly?

Reply:
I suppose we could, but that would be duplicating `dune build src/extension.bc.js` in two places.

Is there was a way for `dune build` to build that JS file automatically?

Reply:
We could introduce an alias. E.g. `dune build @vscode` or even just `$ dune build @all`.